github/ci/prow/files/plugins.yaml

```diff
@@ -66,6 +66,10 @@ plugins:
   - approve
   - lgtm
   - owners-label
+  - config-updater
+  - skip
```

Comment:
I think it makes sense to enable `skip` and `override` for the whole org (the `kubevirt:` section directly below `plugins:`.

Reply:
Make sense, yes. Almost all will end needing it. Adding them to the org section